lib/bot.rb

```diff
@@ -10,8 +10,8 @@
 require "parser/jira_to_github/heading"
 
 class Bot
-  PR_NAME_TICKET_ID_REGEX = /\A\[#LIEF-(\d{3,})\]/
-  BRANCH_NAME_TICKET_ID_REGEX = /\A\w+\/lief-(\d{3,})/
+  PR_NAME_TICKET_ID_REGEX = /\A\[LIEF-(\d{3,})\]/
+  BRANCH_NAME_TICKET_ID_REGEX = /\A\w+\/LIEF-(\d{3,})/
```

Comment:
I think we should use `JIRA_PROJECT_KEY` instead but we should get this merged as soon as possible so I'd prefer improving this in a separate PR.